src/js/view/controls/controlbar.js

```diff
@@ -148,6 +149,10 @@ export default class Controlbar {
         }, localization.liveBroadcast);
         liveButton.element().textContent = localization.liveBroadcast;
 
+        const fullscreenButton = !this._isTizenApp && button('jw-icon-fullscreen', () => {
+            _api.setFullscreen();
+        }, localization.fullscreen, cloneIcons('fullscreen-off,fullscreen-on'));
```

Comment:
Could we just hide the fullscreen button with CSS? Less code and we can split out Tizen css into it's own module/webpack-chunk very easily.

Reply:
Good idea! Makes things a lot simpler.